README.md

```diff
@@ -1,34 +1,111 @@
-# Tutorial on Trixi.jl at ICOSAHOM 2021
+# Trixi.jl: High-Order Numerical Simulations in Julia
 
 This is the companion repository for the tutorial on
 [Trixi.jl](https://github.com/trixi-framework/Trixi.jl) at
 [ICOSAHOM 2021](https://www.icosahom2020.org):
 
-**Tutorial 2: Trixi.jl**
-*Venue:* gather.town (link to be added later)</br>
-*Date & time:* Wednesday, 14th July 2021, 6:15pm - 8:15pm (CEST)</br>
-*Chaired by:* Hendrik Ranocha, Michael Schlottke-Lakemper</br>
+**Tutorial 2: Trixi.jl**  
+*Venue:* gather.town (link to be added later)  
+*Date & time:* Wednesday, 14th July 2021, 6:15pm - 8:15pm (CEST)  
+*Chaired by:* Hendrik Ranocha, Michael Schlottke-Lakemper  
 *Link:* [conference agenda](https://www.conftool.com/icosahom2020/index.php?page=browseSessions&form_session=71&presentations=show)
 
 **Note: This repository is still work-in-progress and will receive further
 updates until the day before the tutorial session.**
 
-In case of questions before the tutorial, please get in touch with
+In case of questions before the beginning of the tutorial, please get in touch with
 [Hendrik](https://ranocha.de) or
 [Michael](https://www.mi.uni-koeln.de/NumSim/schlottke-lakemper),
 [create an issue](https://github.com/trixi-framework/tutorial-2021-icosahom/issues/new),
 or
 [join the Trixi.jl Slack workspace](https://join.slack.com/t/trixi-framework/shared_invite/zt-sgkc6ppw-6OXJqZAD5SPjBYqLd8MU~g).
 
-## Table of contents
-
-| Jupyter notebook | [nbviewer](https://nbviewer.jupyter.org/) | [mybinder](https://mybinder.org/) |
+**Table of contents**
+1. [Schedule](#schedule)
+2. [Floor plan](#floor-plan)
+3. [Tutorial files](#tutorial-files)
+4. [Abstract](#abstract)
+5. [Getting started](#getting-started)
+   1. [Using mybinder.org](#using-mybinderorg)
+   2. [Setting up a local Julia/Jupyter installation](#setting-up-a-local-juliajupyter-installation)
+6. [Authors](#authors)
+7. [License](#license)
+
+
+## Schedule
+
+The [Auditorium](#auditorium) will be used to give a few guided sessions on how
+to use Trixi.jl and Julia. The [Computer Lab](#computer-lab) has work spaces to
+try out Trixi.jl on your own and with the help of our instructors. Please check
+out the [floor plan](#floor-plan) to help with finding out where everthing is
+located.
+
+*Note:* All times are on Central European Summer Time (CEST).
```

Comment:
*in* instead of *on*?

Reply:
I am not sure but yours doesn't sound worse thus fixed in 826514e.